packages/react-vapor/src/components/limit/Limit.tsx

```diff
@@ -0,0 +1,210 @@
+import classNames from 'classnames';
+import * as VaporSVG from 'coveo-styleguide';
+import * as React from 'react';
+import {connect} from 'react-redux';
+import {InputConnected, Svg} from 'react-vapor';
+import {IReactVaporState} from '../../ReactVapor';
+import {IDispatch} from '../../utils/ReduxUtils';
+import {changeInputValue} from '../input/InputActions';
+import {InputSelectors} from '../input/InputSelectors';
+
+export interface LimitOwnProps {
+    id?: string;
+    title: string;
+    limitLabel?: string;
+    usage?: number;
+    limit?: number;
+    isLimitEditable?: boolean;
+    isLimitTheGoalToReach?: boolean;
+    isHistoryIncluded?: boolean;
+}
+
+const mapStateToProps = (state: IReactVaporState, ownProps: LimitOwnProps) => {
+    const ownLimit: string = ownProps.limit.toString() || Limit.defaultProps.limit.toString();
+    const currentLimit: string = InputSelectors.getValue(state, {id: ownProps.id}) || ownLimit;
+    return {
+        currentLimit,
+    };
+};
+
+const mapDispatchToProps = (dispatch: IDispatch, ownProps: LimitOwnProps) => ({
+    onChangeLimit: (limitValue: string) => {
+        dispatch(changeInputValue(ownProps.id, limitValue, true));
+    },
+});
+
+export interface LimitProps
+    extends LimitOwnProps,
+        ReturnType<typeof mapDispatchToProps>,
+        ReturnType<typeof mapStateToProps> {}
+
+export const LimitDisconnect: React.FunctionComponent<LimitProps> = ({
```

Comment:
We generally don't export the unconnected version.
```suggestion
const LimitDisconnect: React.FunctionComponent<LimitProps> = ({
```